src/Blocks/WorldInterface.h

```diff
@@ -61,7 +61,7 @@ class cWorldInterface
 	virtual void SendBlockTo(int a_BlockX, int a_BlockY, int a_BlockZ, cPlayer & a_Player) = 0;
 
 	/** Sends the block on those coords to the player */
-	inline void SendBlockTo(const Vector3i a_BlockPos, cPlayer & a_Player)
+	virtual inline void SendBlockTo(const Vector3i a_BlockPos, cPlayer & a_Player)
```

Comment:
Nah, only one overload needs to be virtual.

Reply:
Waht do you meen? Whe I remove the virtual expression we're back to square one and can't explicitly overide the function

Reply:
In fact I think you can delete the body of this altogether and make it pure virtual

Reply:
`SendBlockTo(int, int, int)` can and should do the exact same thing as `SendBlockTo(Vector3i)`, so there's no reason for this to be virtual.

Reply:
But the method in cWorld shadows this method in cWorldInterface. This was the reason for this PR